checks/data_platform/live_main_ping_age.py

```diff
@@ -0,0 +1,39 @@
+"""
+Age, in seconds, of the latest live main ping measured as the time since the submission_timestamp
+of the ping.  This is meant to be used as an approximation of the time it takes for pings to
+travel from the edge server to the live tables.
+
+Return failure if more than `max_error_rate` of the last `value_count` values
+are above `max_threshold`
+"""
+from poucave.typings import CheckResult
+from poucave.utils import fetch_redash
+
+
+REDASH_QUERY_ID = 69148
+
+
+async def run(
+    api_key: str,
+    max_threshold: float = 700,
```

Comment:
you can expose this parameter if necessary. See https://github.com/mozilla-services/poucave/blob/e605f2bf0a72952ee562931246161e554f3bbb81/checks/remotesettings/certificates_expiration.py#L23